plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -212,6 +212,11 @@ public String toString() {
 		}
 
 		@Override
+		public void widgetDefaultSelected(SelectionEvent e) {
+			widgetDefaultSelected(e);
+		}
```

Comment:
you're right in noticing that the above patch is actually an endless loop and obviously cant be anything of a fix. I changed the above to a few variants like super.widgetDefaut..(), widgetSelected() etc.
But all this has 0 effect. #widgetDefaultSelected never gets called